projects/ngneat/transloco/src/lib/tests/transloco.transpiler.spec.ts

```diff
@@ -63,19 +68,27 @@ describe('TranslocoTranspiler', () => {
     });
   });
 
-  function testDefaultBehaviour(parser: TranslocoTranspiler) {
+  function testDefaultBehaviour(parser: TranslocoTranspiler, interpolationMarkings: [string, string] = ['{{', '}}']) {
     it('should translate simple string from params', () => {
-      const parsed = parser.transpile('Hello {{ value }}', { value: 'World' }, {});
+      const parsed = parser.transpile(`Hello ${wrapParam('value', interpolationMarkings)}`, { value: 'World' }, {});
```

Comment:
why do you need to pass the interpolationMarkings to the function? just create it in the testDefaultBehaviour scope.